src/actions/conversation.js

```diff
@@ -62,6 +79,51 @@ export const getConversations = ({
   }
 };
 
+// Load more  conversations
+export const loadMoreConversation = ({
+  assigneeType,
+  conversationStatus,
+  inboxSelected,
+  pageNumber,
+}) => async dispatch => {
+  dispatch({ type: GET_MORE_CONVERSATIONS });
+  try {
+    let assignee;
+    switch (assigneeType) {
+      case 0:
+        assignee = ASSIGNEE_TYPE.ME;
+        break;
+      case 1:
+        assignee = ASSIGNEE_TYPE.UN_ASSIGNED;
+        break;
+      default:
+        assignee = ASSIGNEE_TYPE.ALL;
+    }
+
+    const status = conversationStatus === 'Open' ? 'open' : 'resolved';
+    const inboxId = inboxSelected && inboxSelected ? inboxSelected.id : null;
+    const apiUrl = `${API}conversations?${
```

Comment:
Same why do we need an additional action to load more conversation, both cases should be handled in one action